tests/php/Extension/FluentSiteTreeExtensionTest.yml

```diff
@@ -54,22 +54,3 @@ Page:
     Title: Staff
     URLSegment: my-staff
     Parent: =>Page.about
-
-SiteTree_Localised:
-  nz-page:
-    RecordID: =>Page.nz-page
-    Title: A Page
-    URLSegment: a-page
-    Locale: de_DE
-  about:
-    RecordID: =>Page.about
-    Title: About Us
-    URLSegment: about-us
-    Locale: de_DE
```

Comment:
Removing this is necessary for `FluentSiteTreeExtensionTest::testUpdateCMSActionsDrafted()` to pass - see failure in https://github.com/tractorcow-farm/silverstripe-fluent/actions/runs/17311586305/job/49146679097

This fixture was adding incomplete localisation data to the localised table. Removing it allows fluent to build the correct localisation data and means we're testing the _actual_ functionality rather than just testing whether the fixture has the right localisation data or not.

Removing the fixture data for `nz-page` isn't strictly necessary for _this_ PR but it should be removed for the same reasons.